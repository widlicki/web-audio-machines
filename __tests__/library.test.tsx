import { render } from '@testing-library/react'
import Library from '../pages/library'

describe('Library', () => {
  it('renders library page according to snapshot', () => {
    const { container } = render(<Library />)
    expect(container).toMatchSnapshot()
  })
})
