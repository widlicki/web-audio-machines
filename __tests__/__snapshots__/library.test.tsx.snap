// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Library renders library page according to snapshot 1`] = `
<div>
  <div>
    <h1>
      Library
    </h1>
  </div>
</div>
`;
